.github/workflows/docs-deploy.yml

```
@@ -81,6 +81,13 @@ jobs:
           ALGOLIA_SEARCH_API_KEY: ${{ secrets.ALGOLIA_SEARCH_API_KEY }}
           GA_TRACKING_ID: ${{ secrets.GA_TRACKING_ID }}
 
+      - name: Push search index to Algolia
+        if: ${{ secrets.ALGOLIA_APP_ID != '' && secrets.ALGOLIA_ADMIN_API_KEY != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)